src/Components/Button/Button.jsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 
 import "./Button.scss";
 
+import clsx from "clsx";
```

Comment:
could you also add clsx to the first line of the prettier imports config?

Reply:
done